openedx/core/djangoapps/discussions/docs/decisions/0005-in-context-discussion-course-blocks.rst

```diff
@@ -0,0 +1,39 @@
+Making Linked In-Context Discussions Available to MFEs
+======================================================
+
+
+Status
+------
+
+Proposal
+
+
+Context
+-------
+
+In the ADR 0004 we investigated how to link a discussion to a unit. We
+also need some way to make these linked discussions available to the
+frontend so they can display those in-context discussions in the correct
+view.
+
+
+Requirements
+------------
+
+An API to access linked discussions for a Unit.
+
+
+Decision
+--------
+
+A direct link to the topic that needs to be embedded can be generated
+by edx-platform and provided to MFEs via the course blocks API which is
+already is used by the learning MFE. The learning MFE can then directly
+embed this link in a sidebar.
+
+For units that don't have a linked discussion, no link will be returned.
+
+If the setting to group discussions at the subsection level is enabled,
+then this is handled by the discussions MFE, and the parent topic that
+rolls up all discussions in that subsection will be provided as the
+embedded URL instead.
```

Comment:
This paragraph is not entirely clear to me. What is `setting to group discussions`? I think this paragraph needs elaboration.

Reply:
I've expanded on this.